ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -14,7 +14,12 @@ public static string GetNotificationEditPath(string subPath)
 
         public static string GetFullNotificationPath(string subPath, int id)
         {
-            return $"/Notifications/{subPath}?id={id}";
+            return $"{GetNotificationPath(subPath)}?id={id}";
+        }
+
+        public static string GetNotificationPath(string subPath)

```

Comment:
I'm surprised this is useful without the id ... it's not really a valid path by itself?
If we were to move to setting `id` as path parameter (instead of query param) this will prove problematic I think?

Reply:
I'm calling those directly from Routes.cs in integration tests so I can name the base routes and use them as static string, and then generate the full route in the test.. I can call it GetNotificationBasePath to make it clear it shouldn't be used by itself?

Reply:
Ah, gotcha. yes, that's clearer thanks. i think it'll still come up if we change the url structure, but we can worry about exactly what it should look like then :)
thanks!